ppcs/ppc0019-qt-string.md

```diff
@@ -70,6 +70,38 @@ will be equivalent to
 qq{A ${ \scalar(TEXT) } B};
 ```
 
+Escape sequences that are recognised in `qq` strings as standing in place for
+other literal characters (such as `\n` for linefeed or `\x00` for a NUL byte)
```

Comment:
I'm not sure I like calling `\n` a "linefeed". Do we do that in other places, too?

(Perl borrows `\n` from C's I/O model, which has no linefeed, only newline.)

Reply:
I would reword this to avoid the passive voice. This puts the most important information at the front.

> `qt` also recognizes the escape sequences used in `qq` strings as representing
other literal characters (such as `\n` for newline or `\x00` for a NUL byte)

Reply:
Ahyes, sloppy wording on my part there. perlop(1) calls it "newline" so I should do the same.